esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -3153,8 +3176,35 @@ bool CFileSprayEx::onDropZoneFileSearch(IEspContext &context, IEspDropZoneFileSe
         if (!validNameFilter)
             throw MakeStringException(ECLWATCH_INVALID_INPUT, "Invalid Name Filter '*'");
 
+        double version = context.getClientVersion();
+        bool serverFound = false;
+        unsigned filesFound = 0;
+        IArrayOf<IEspPhysicalFileStruct> files;
 #ifdef _CONTAINERIZED
-        UNIMPLEMENTED_X("CONTAINERIZED(CFileSprayEx::onDropZoneFileSearch)");
+        bool isIPAddressReq = isIPAddress(dropZoneServerReq);
+        ForEachItemIn(i, allTpDropZones)
+        {
+            IConstTpDropZone& dropZone = allTpDropZones.item(i);
+            const char* name = dropZone.getName();
+            if (!streq(dropZoneName, name))
+                continue;
+            if (req.getECLWatchVisibleOnly() && !dropZone.getECLWatchVisible())
+                continue;
+
+            IArrayOf<IConstTpMachine>& tpMachines = dropZone.getTpMachines();
+            ForEachItemIn(ii, tpMachines)
+            {
+                IConstTpMachine& tpMachine = tpMachines.item(ii);
+                const char* netAddress = tpMachine.getNetaddress();
+                if (isEmptyString(netAddress) || !matchNetAddressRequest(dropZoneServerReq, isIPAddressReq, tpMachine))
```

Comment:
curious: What is the check for an empty netAddress for?

Reply:
The netAddress is used in line 3203. So, it is checked here.